app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -210,31 +211,42 @@ fun SyncedTabsSectionHeader(
  */
 @Composable
 fun SyncedTabsTabItem(tabTitleText: String, url: String, onClick: () -> Unit) {
-    Column(
+    Row(
         modifier = Modifier
             .clickable(
                 onClickLabel = tabTitleText,
                 onClick = onClick
             )
             .padding(horizontal = 16.dp)
             .defaultMinSize(minHeight = 56.dp),
-        verticalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
     ) {
-        PrimaryText(
-            text = tabTitleText,
-            modifier = Modifier.fillMaxWidth(),
-            fontSize = 16.sp,
-            maxLines = 1
+        Favicon(
```

Comment:
The `@Preview` is broken from using `Favicon()`, so I'll be making a follow-up issue to refactor it to allow for previewing.